dts/bindings/pinctrl/renesas,ra-pinctrl.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Renesas RA series pin controller
+
+compatible: "renesas,ra-pinctrl"
+
+include: base.yaml
+
+child-binding:
+  description: |
+    Definitions for a pinctrl state.
+  child-binding:
+
+    include:
+      - name: pincfg-node.yaml
+        property-allowlist:
+          - bias-disable
+          - bias-pull-up
+          - drive-open-drain
+          - input-enable
+          - drive-strength
```

Comment:
where are these handled? I can't see them being referenced in pinctrl_soc.h

Reply:
Not yet implemented these, but the reference manual says, these functions exist.
I am adding these for now as I will implement them in the near future.
Is it better not to enter it at this time?
![ra4m1_pinconf](https://github.com/zephyrproject-rtos/zephyr/assets/458281/6053400d-8984-4248-a4b3-90b3fe057b8a)

Reply:
If they're not used, they shouldn't be allowed yet, this will create confusion to the users.

Reply:
Ah, I got it.
I removed it.